Auth0/WebAuth.swift

```diff
@@ -200,4 +200,20 @@ public protocol WebAuth: Trackable, Loggable {
      - parameter callback: callback called with the result of the WebAuth flow
      */
     func start(_ callback: @escaping (Result<Credentials>) -> Void)
+
+    /**
+     Clear the SSO Cookie in Auth0 and optionally signout from IdP.
```

Comment:
Removes Auth0 session and optionally allows to remove the Identity Provider session.